canopen/source/co_sync.c

```diff
@@ -57,6 +62,7 @@ void COSyncInit(CO_SYNC *sync, struct CO_NODE_T *node)
     if (err != CO_ERR_NONE) {
         node->Error = CO_ERR_CFG_1005_0;
         sync->CobId = 0;
+        return;
```

Comment:
Is this return statement a relict from some tests? I think this return is not needed.